tests/integration_tests/functional/test_mmds.py

```diff
@@ -28,30 +28,6 @@ def test_mmds(test_microvm_with_ssh, network_config):
     assert test_microvm.api_session.is_status_ok(response.status_code)
     assert response.json() == {}
 
-    # Test that patch return NotFound when the MMDS is not initialized.
```

Comment:
I like it better with one test/resource. Is one of the python linters complain about too many statements?

Reply:
Yes, it complained that the number of statements was over 50.

Reply:
In this case you will need to squash this commit and the following 2 into one. The next two commits are there to fix this one from what I understand.

Reply:
Also, you have a typo in the commit message (statesments).

Reply:
Thanks! I squashed and fix typo.